app/src/main/java/de/danoeh/antennapod/dialog/ItemFilterDialog.java

```diff
@@ -54,6 +58,41 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
             rows.addView(binding.getRoot());
         }
 
+        //add confirm and reset button
+        MaterialButton confirmButton = new MaterialButton(getContext());
+        confirmButton.setText(getString(R.string.confirm_label));
+        confirmButton.setOnClickListener(view1 -> {
+            dismiss();
+        });
+        MaterialButton resetButton = new MaterialButton(getContext());
+        resetButton.setText(getString(R.string.reset));
+        resetButton.setOnClickListener(view1 -> {
+            onFilterChanged(Collections.emptySet());
+            for (int i = 0; i < rows.getChildCount(); i++) {
+                if (rows.getChildAt(i) instanceof MaterialButtonToggleGroup) {
+                    ((MaterialButtonToggleGroup) rows.getChildAt(i)).clearChecked();
+                }
+            }
+            dismiss();
+        });
+        float eightDp = TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, 8,
```

Comment:
Please specify the buttons in xml instead, that's a lot easier. The relevant file would be `filter_dialog.xml`. In xml, you can then use `style="@style/Widget.MaterialComponents.Button.TextButton"` to get the normal text button design that AlertDialogs use as well.

Reply:
I just saw your edit to the previous comment. Using the style suggested above keeps the touch target large, even if the button is then text-only. Just like the confirmation button on alert dialogs. That takes away the strong visual focus that the buttons currently have in your design.